frontend/components/TargetLabelSelector/TargetLabelSelector.tsx

```diff
@@ -136,14 +138,18 @@ const LabelChooser = ({
 
   return (
     <div className={`${baseClass}__custom-label-chooser`}>
-      <Dropdown
-        value={selectedCustomTarget}
-        options={customTargetOptions}
-        searchable={false}
-        onChange={onSelectCustomTarget}
-      />
+      {customTargetOptions.length ? (
+        <Dropdown
```

Comment:
Can we `&&` instead of ternary with `null`?

Reply:
Done!